pkg/kubelet/hyper/hyperclient.go

```diff
@@ -141,6 +141,14 @@ type podRemoveResult struct {
 	ID    string `json:"ID"`
 }
 
+type HyperPod struct {
```

Comment:
Why not use HyperPod directly just like other functions did? `hyper/types.go` and this belongs to the same package ...